docs/project_overview.md

```diff
@@ -11,6 +11,7 @@ to check out the files themselves and read the docstrings.
 
 Used for management of the Git repository. It specifies which files should not be uploaded
 to the repository when you make your Git commits.
+NOTE: This file tells git what to ignore, and as a contributor you do not need to worry about it, git will make sure the right things are ignored. 
```

Comment:
This is a very small point, but grammatically, I think this needs to be split up into two sentences. Also, I feel like you don't _really_ need the "NOTE:" at the beginning.

```
This file tells Git what to ignore, and as a contributor you do not need to worry about it. Git will make sure the right things are ignored.
```

But this is a very subjective suggestion.